e2e/commands_test.go

```diff
@@ -58,8 +66,14 @@ func testRenderApp(appPath string, env ...string) func(*testing.T) {
 		}
 		cmd.Command = args
 		cmd.Env = append(cmd.Env, env...)
+		// Check rendering to stdout
 		result := icmd.RunCmd(cmd).Assert(t, icmd.Success)
 		assert.Assert(t, is.Equal(readFile(t, filepath.Join(appPath, "expected.txt")), result.Stdout()), "rendering mismatch")
+		// Checks rendering to a file
+		cmd.Command = append(cmd.Command, "--output="+dir.Join("actual.yaml"))
+		result = icmd.RunCmd(cmd).Assert(t, icmd.Success)
+
+		assert.Assert(t, is.Equal(readFile(t, filepath.Join(appPath, "expected.txt")), readFile(t, dir.Join("actual.yaml"))), "rendering mismatch")
```

Comment:
nit: that blank line seem spurious...

this looks like an opportunity to use `golden` but since you are following the existing idiom in this file lets not worry about that.

Reply:
I tried then reverted an attempt to use `golden`, but there is a trick with the filepath, as golden append itself the directory `testdata` to the path, and the resulting code was really ugly...

Reply:
Ah, I see that the path to `expected.txt` is a bit dynamic. That makes sense then. Thanks.